src/adapters/User/index.ts

```diff
@@ -0,0 +1,3 @@
+import { getAuth } from 'adapters/BaseAuth';
+
+export const getUserInfo = () => getAuth('me');
```

Comment:
is it supposed to be?
```suggestion
export const getUserInfo = async (): Promise<User> => getAuth('me');
```

Reply:
@nvminhtue the return type is `Promise<AxiosResponse<any, any>>`, we haven't deserialized it to `User` yet. So I think we can write it shorter like above, or:
```suggestion
export const getUserInfo = (): Promise<AxiosResponse<any, any>> => getAuth('me');
```